UIforETW/DirectoryMonitor.cpp

```diff
@@ -62,21 +61,23 @@ DWORD DirectoryMonitor::DirectoryMonitorThread()
 			return 0;
 
 		default:
-			assert(0);
+			ATLASSERT(0);
 			break;
 		}
 	}
   // Unreachable.
 }
 
+_Pre_satisfies_(this->hThread_ == 0)

```

Comment:
Yeah, `_In_` (which GitHub "helpfully" formats if not enclosed in backticks) is kinda useless - it'd be nice if it did something like imply `_Const_`, or otherwise check access, but *nope* it doesn't.

I'm guessing that it (specifically doesn't) is an artifact of Microsoft using `SALInfer` on their entire codebase, which I can imagine producing 1,000,000+ false positives.